src/uid2HashedDii.ts

```diff
@@ -0,0 +1,13 @@
+export type Base64Hash = string;
+
+export function isBase64Hash(value: string): value is Base64Hash {
+  if (!(value && value.length === 44)) {
+    return false;
+  }
+
+  try {
+    return btoa(atob(value)) === value;
```

Comment:
vscode is telling me that btoa and atob are both deprecated - it is saying that `buf.toString('base64')` and `Buffer.from(data, 'base64')` should be used instead - could we do that?

Reply:
Unfortunately no, `Buffer` is specific to Node.js.

Reply:
Generally speaking, if it's running in the browser it's not Node.js. Node.js is for server-side or desktop apps.